rust-runtime/aws-smithy-client/src/hyper_ext.rs

```diff
@@ -252,7 +252,7 @@ impl Builder {
     }
 }
 
-#[cfg(any(feature = "rustls", feature = "native_tls"))]
+#[cfg(any(feature = "rustls", feature = "native-tls"))]
```

Comment:
This was a really annoying thing to solve, I wish the compiler had assisted me here

Reply:
I bet you could add a clippy lint for features that don't exist...

Reply:
I spoke to Esteban about it and he advised me to file [this issue](https://github.com/rust-lang/rust/issues/91674)